src/components/AppLayout/Header/components/ProviderDetails/UserDetails.tsx

```diff
@@ -97,6 +99,7 @@ type Props = {
   connected: boolean
   network: ETHEREUM_NETWORK
   onDisconnect: () => void
+  onNetworkChange: () => any
```

Comment:
```suggestion
  onNetworkChange: () => Promise<void>
```

Reply:
This was actually intentional. When we pass a callback to a component that doesn't care about the return value, we don't want to be specific about its return value. It can take anything.

Reply:
`unknown` would work too, wouldn't it? Just with some additional type safety